components/table/html.scss

```diff
@@ -25,7 +25,7 @@
 }
 
 .utrecht-html th {
-  @extend .utrecht-table__heading;
+  @extend .utrecht-table__header-cell;
```

Comment:
What's the reasoning for adding the `-cell` part? `<th>` already indicates a single header. This kind of feels as though there is also `table-header` component. (Asking because we went for `table-header` with [Den Haag](https://github.com/nl-design-system/denhaag/blob/main/components/Table/src/TableHeader.tsx)).

Reply:
I went with `table__header`/`table__body`/`table__footer` for the `thead`/`tbody`/`tfoot` sections, so `header` was indeed already taken.

@Yolijn any thoughts?